docs/freqtrade-101.md

```diff
@@ -0,0 +1,188 @@
+
+Examples of common indicators include Relative Strength Index, Bollinger Bands, Money Flow Index, Moving Average, and Average True Range.
+
+Columns are added to the dataframe by calling technical analysis functions, e.g. ta-lib's RSI function `ta.RSI()`, and assigning them to a column name, e.g. `rsi`
+
+```python
+dataframe['rsi'] = ta.RSI(dataframe)
+```
+
+??? Hint "Technical Analysis libraries"
+    Different libraries work in different ways to generate indicator values. Please check the documentation of each library to understand
+    how to integrate it into your strategy. You can also check the [Freqtrade example strategies](https://github.com/freqtrade/freqtrade-strategies) to give you ideas.
+
+### Populate entry signals
+
+The `populate_entry_trend` function defines conditions for an entry signal.
+
+The dataframe column `enter_long` is added to the dataframe, and when a value of `1` is in this column, Freqtrade sees an entry signal.
+
+??? Hint "Shorting"
+    To enter short trades, use the `enter_short` column.
+
+### Populate exit signals
+
+The `populate_exit_trend` function defines conditions for an exit signal.
+
+The dataframe column `exit_long` is added to the dataframe, and when a value of `1` is in this column, Freqtrade sees an exit signal.
+
+??? Hint "Shorting"
+    To exit short trades, use the `exit_short` column.
+
+## A simple strategy
+
+Here is a minimal example of a Freqtrade strategy:
+
+```python
+from freqtrade.strategy.interface import IStrategy
+from pandas import DataFrame
+import talib.abstract as ta
+
+class MyStrategy(IStrategy):
+
+    def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+        # generate values for technical analysis indicators
+        dataframe['rsi'] = ta.RSI(dataframe, timeperiod=14)
+
+        return dataframe
+
+    def populate_entry_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+        # generate entry signals based on indicator values
+        dataframe.loc[
+            (dataframe['rsi'] < 30),
+            'enter_long'] = 1
+
+        return dataframe
+
+    def populate_exit_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+        # generate exit signals based on indicator values
+        dataframe.loc[
+            (dataframe['rsi'] > 70),
+            'exit_long'] = 1
+
+        return dataframe
+```
+
+## Making trades
+
+When a signal is found (a `1` in an entry or exit column), Freqtrade will attempt to make an order, i.e. a `trade` or `position`.
+
+The number of concurrent trades that can be opened is defined by the `max_open_trades` [configuration](configuration.md) option.
+
+However, there can be a range of scenarios where generating a signal does not always create a trade order. These include:
+
+- not enough remaining stake to buy an asset, or funds in your wallet to sell an asset (including any fees)
+- not enough open slots for a new trade to be opened
```

Comment:
(reading with an unknowing mind ...) 
Above we talked about "concurrent trades" - so what's slots again?